[stable/owncloud] Set deployment strategy with values

zabudskyi opened the pull request:
Signed-off-by: Pavlo Zabudskyi <pavlo.zabudskyi@gmail.com>

#### What this PR does / why we need it:
We need possibility to set `.spec.strategy.type` to `Recreate` for owncloud deployment. When PVC access mode is set to `ReadWriteOnce` and pod is deleted, it can't re-map volume. Default value is set to `RollingUpdate`.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

clamoriniere committed:
[stable/datadog] fix dd_site and dd_dd_url env var (#19913)

Fix the usage of the `.Values.datadog.site` and `.Values.datadog.dd_url`
parameters when `.Values.daemonset.useDedicatedContainers` is activated.

Signed-off-by: cedric lamoriniere <cedric.lamoriniere@datadoghq.com>

Smirl committed:
[stable/prometheus-operator] - add tlsConfig to prometheus servicemonitor (#19848)

* prometheus-operator - add tlsConfig to prometheus servicemonitor

Signed-off-by: Alex Williams <alex.williams@skyscanner.net>

* Bump to 8.5.5.

Signed-off-by: Alex Williams <alex.williams@skyscanner.net>

* nindent on correct line

Signed-off-by: Alex Williams <alex.williams@skyscanner.net>

Commit:
[stable/suitecrm] Release 8.0.2 updating components versions (#19980)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

monotek committed:
[stable/prometheus] update prometheus to 2.15.2 and also all other used images to newest … (#19927)

* update prometheus to 2.15.2 and also all other used images to newest version

Signed-off-by: André Bauer <monotek23@gmail.com>

* changed alertmanager baseurl to satisfy the new check of the url on container startup

Signed-off-by: André Bauer <monotek23@gmail.com>

* removed executable flag from files

Signed-off-by: André Bauer <monotek23@gmail.com>

* fix merge conflict

Signed-off-by: André Bauer <andre.bauer@kiwigrid.com>

maruina committed:
[stable/chartmuseum] Add support for LoadBalancer and loadBalancerSourceRanges (#19967)

* Add support for loadBalancerSourceRanges

Signed-off-by: Matteo Ruina <matteo.ruina@skyscanner.net>

* Fix indentation

Signed-off-by: Matteo Ruina <matteo.ruina@skyscanner.net>

* Bump minor version

Signed-off-by: Matteo Ruina <matteo.ruina@skyscanner.net>

dawidmalina committed:
Revert - File system 'fs-12345678' does not exist (#19984)

Signed-off-by: Dawid Malinowski <dawidmalina@gmail.com>

cgiraldo committed:
[incubator/schema-registry] Deployment extensions/v1beta1 to apps/v1 (#19983)

* Deployment extensions/v1beta1 to apps/v1

Deployment in extensions/v1beta1 API group is removed in kubernetes v1.16.
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Changed Deployment to apps/v1 API group.

Signed-off-by: Carlos Giraldo <cgiraldo@gradiant.org>

* Chart version bumped

Signed-off-by: Carlos Giraldo <cgiraldo@gradiant.org>

yujin-hong committed:
[stable/prometheus-operator] fix typo for prometheus Operator Service… (#19969)

* Bump prometheus-operator chart version

Signed-off-by: yujin-hong <dbwlsdl2500@gist.ac.kr>

* [stable/prometheus-operater] fix typo for prometheus Operator Service type comment

Service type `NodePort` was denoted as `NodepPort`. It can cause confusion to beginners.

Signed-off-by: yujin-hong <dbwlsdl2500@gist.ac.kr>

Commit:
[stable/cockroachdb]: Update version to v19.2.2 (#19990)

Signed-off-by: Jesse Seldess <j_seldess@hotmail.com>

thewilli committed:
[stable/minio] fix handling of extraArgs config (#19904)

starting with v3.0.0 of this Helm Chart, the
extraArgs configuration value stopped working,
as it was not passed through the Minio binary.

This fix adds the parameters directly to the
Entrypoint command.

Fixes #19903

Signed-off-by: Willi Eggeling <thewilli@gmail.com>

Commit:
[stable/graylog] Update mongodb-replicaset dependency version. (#20005)

Fixes #18997, which causes graylog chart to fail due to deprecated
API version used by the old mongodb-replicaset chart.

Signed-off-by: juliohm1978 <jhm@juliohm.com.br>

Commit:
[stable/owncloud] Release 8.0.2 updating components versions (#19989)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/dokuwiki] Release 6.0.3 updating components versions (#19993)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/opencart] Release 7.0.2 updating components versions (#20008)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

vishal-vp committed:
Fix typo in mongodb README (#20006)

Signed-off-by: vishal-vp <21176895+vishal-vp@users.noreply.github.com>

Commit:
[stable/external-dns] Release 2.14.1 updating components versions (#20007)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

lawliet89 committed:
[stable/prometheus] Add Missing ClusterRole Permissions for Kube-state-metrics (#20004)

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

ta-ando committed:
Add values of paths for livenessProbe and readinessProbe (#20011)

Signed-off-by: Takashi Ando <ta.ando@lab.acs-jp.com>

Commit:
[stable/mediawiki] Release 9.1.1 updating components versions (#20010)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/redis] Release 10.3.2 updating components versions (#20015)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

mrestivill committed:
[stable/tomcat] Added failureThreshold on livenessProbe and timeoutSeconds (#19214)

* [stable/pgadmin] Update image v4.14 to v4.15 (#19028)

* [stable/pgadmin] Update image v4.14 to v4.15

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

* chart version bump

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>
Signed-off-by: Marc Rodriguez-Estivill <mrodriguezestivill@gmail.com>

* Added failureThreshold on livenessProbe

Signed-off-by: Marc Rodriguez-Estivill <mrodriguezestivill@gmail.com>

* bump version to 0.4.1

Signed-off-by: Marc Rodriguez-Estivill <mrodriguezestivill@gmail.com>

* Revert "[stable/pgadmin] Update image v4.14 to v4.15 (#19028)"

This reverts commit a23341e868e0cc06cd7f23a2641823dc94aad529.

Signed-off-by: Marc Rodriguez-Estivill <mrodriguezestivill@gmail.com>

* added timeoutSeconds

Signed-off-by: Marc Rodriguez-Estivill <mrodriguezestivill@gmail.com>

Co-authored-by: rowanr <rowanruseler@gmail.com>

pauvos committed:
[stable/drone] drone server and restricted pod security policies (#19712)

* [stable/drone] add securityContext to drone server

prevents "Error: container has runAsNonRoot and image will run as root"
in k8s clusters with podSecuritypolicies

Signed-off-by: Paul Voss <paul@paulvoss.net>

* [stable/drone] add setting for the drone server http port

allows running the server pod as non-root user

Signed-off-by: Paul Voss <paul@paulvoss.net>

* [stable/drone] update README.md and bump chart version

Signed-off-by: Paul Voss <paul@paulvoss.net>

* [stable/drone] bump Chart version to 2.5.0

Signed-off-by: Paul Voss <paul@paulvoss.net>

wilmardo committed:
fix: reverts changes to sentry secrets template (#20018)

Signed-off-by: wilmardo <info@wilmardenouden.nl>

morganchristiansson committed:
[stable/redis-ha] add persistentVolume.reclaimPolicy (#19912)

* [stable/redis-ha] add persistentVolume.reclaimPolicy

Also document extra persistentVolume variables and reformat markdown table

Signed-off-by: Morgan Christiansson <git@mog.se>

* [stable/redis] Change persistentVolume.reclaimPolicy default to nil

Thanks @DandyDeveloper

Signed-off-by: Morgan Christiansson <git@mog.se>
Co-Authored-By: Aaron Layfield <Aaron.Layfield@gmail.com>

Co-authored-by: Aaron Layfield <Aaron.Layfield@gmail.com>

Commit:
[stable/prestashop] Release 9.1.1 updating components versions (#20031)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/testlink] Release 7.0.2 updating components versions (#20020)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/ghost] Release 9.1.1 updating components versions (#20014)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/moodle] Release 7.0.5 updating components versions (#20016)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/orangehrm] Release 7.0.2 updating components versions (#20023)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

travisgroth committed:
[stable/pomerium] Allow service and deployment specific annotations (#20003)

Signed-off-by: Derrick Burns <derrick@tidepool.org>

paprickar committed:
[stable/prometheus-pushgateway] Add persistence support (#16040)

* Add persistence support

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* fix linting issue

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* Update Chart version

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* Avoid breaking existing behaviour

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* [stable/prometheus-pushgateway] Bump version (#16040)

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

jieyu committed:
[promethesus-adapter] use the new tag name (#19961)

Some metrics labels (namely `pod_name` and `container_name`) have been
removed in Kubernetes 1.16 (deprecated for a few releases). This patch
switch to use the new metrics labels so that metrics can be properly
retrieved and parsed from Prometheus.

See more details about the removal of metrics labels here:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#removed-metrics

Also, bump the min supported Kubernetes version to 1.14 because the new
metrics labels are introduced in 1.14. See the following release notes:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md

Signed-off-by: Jie Yu <yujie.jay@gmail.com>

Commit:
[stable/sysdig] Remove auditLog.clusterIP value (#20041)

We don't need to use clusterIP in advance:

* If using webhook, we can deploy the agent and then configure webhook
  to post audit logs to the clusterIP.
* If using AuditSink, we can rely on KubeDNS for reaching the service.

Signed-off-by: Néstor Salceda <nestor.salceda@sysdig.com>

jlentzfl committed:
Fix for [stable/kong] servicemonitor.yaml: error converting YAML to JSON #19956 (#19992)

Signed-off-by: Jennifer Lentz <jlentz@finishline.com>

Commit:
[stable/kubewatch] Release 1.0.2 updating components versions (#20036)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/rabbitmq] Release 6.16.1 updating components versions (#20037)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/redmine] Release 14.1.2 updating components versions (#20042)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/postgresql] Release 8.1.3 updating components versions (#20038)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/nats] Release 4.2.8 updating components versions (#20043)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/osclass] Release 7.0.2 updating components versions (#20044)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/external-dns] Release 2.14.2 updating components versions (#20045)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/moodle] Release 7.0.6 updating components versions (#20049)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

sstarcher committed:
workaround ELB healthcheck ssl bug (#20039)

https://github.com/kubernetes/kubernetes/issues/83070

Signed-off-by: Shane Starcher <shanestarcher@gmail.com>

mrueg committed:
[stable/kube-state-metrics] Version bump to 1.9.1 (#20028)

https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.9.1

Signed-off-by: Manuel Rüger <manuel@rueg.eu>

krish7919 committed:
[stable/kong] Add check for config changes (#20051)

- Add an admission webhook config checksum as an annotation in the
Deployment so that the pods can be redeployed by K8s when the config
changes during redeployments

Signed-off-by: Krish <kriiyer@cisco.com>

cgiraldo committed:
[stable/schema-registry-ui] Deployment apps/v1beta1 to apps/v1 (#20012)

* Deployment apps/v1beta1 to apps/v1

Signed-off-by: Carlos Giraldo <cgiraldo@gradiant.org>

* Chart Version bumped

Signed-off-by: Carlos Giraldo <cgiraldo@gradiant.org>

Commit:
[stable/fluent-bit]add use_journal (#19622)

* [stable/fluent-bit] allow collecting audit logs (#19598)

* [fluent-bit] allow collecting audit logs

Signed-off-by: alejandroEsc <jaescobar.cell@gmail.com>

* fixed extranentry item

Signed-off-by: alejandroEsc <jaescobar.cell@gmail.com>
Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* [stable/pomerium] Fix helm lint --strict errors (#19527)

* [stable/pomerium] Added the extra values that are needed to pass helm lint --strict

Signed-off-by: Alexios Polyzos <a.polyzos@thebeat.co>

* [stable/pomerium] Added the extra variables to the Pomerium configuration list

Signed-off-by: Alexios Polyzos <a.polyzos@thebeat.co>

* [stable/pomerium] Bump chart version

Signed-off-by: Alexios Polyzos <a.polyzos@thebeat.co>

* [stable/pomerium] Default name overrides to empty string

Signed-off-by: Alexios Polyzos <a.polyzos@thebeat.co>
Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* [stable/prometheus-blackbox-exporter] Able to mount extra secrets int… (#19119)

* [stable/prometheus-blackbox-exporter] Able to mount extra secrets into the Pod, for example certificates.
Signed-off-by: Hung Do <hdo@infodation.nl>

* [stable/prometheus-blackbox-exporter] Able to mount extra configmaps

Signed-off-by: Hung Do <hdo@infodation.nl>

* [stable/prometheus-blackbox-exporter] Bump up minor version and configmap and secret variables are on par with each other

Signed-off-by: Hung Do <hdo@infodation.nl>

* [stable/prometheus-blackbox-exporter] Improved configmap/secrets examples in the values.yaml

Signed-off-by: Hung Do <hdo@infodation.nl>
Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: Robert Snyder <rsnyder_co@msn.com>
Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: Robert Snyder <rsnyder_co@msn.com>
Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: rss5mpk@ssh://github.homedepot.com <robert_snyder@homedepot.com>

* added use_journal. Signed-off-by: Robert Snyder <rsnyder_co@msn.com>

Signed-off-by: Robert Snyder <robert_snyder@homedepot.com>

Co-authored-by: Alejandro Escobar <jaescobar.cell@gmail.com>
Co-authored-by: Travis Groth <travisgroth@users.noreply.github.com>
Co-authored-by: Terentio <35747908+Terentio@users.noreply.github.com>

dunglas committed:
[stable/mercure] Add support for new features (#19761)

Signed-off-by: Kévin Dunglas <dunglas@gmail.com>

Commit:
[stable/mariadb] Release 7.3.2 updating components versions (#20053)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

secret104278 committed:
[stable/nginx-ingress] Remove duplicated hostPorts entries in psp (#20063)

There are duplicated hostPorts entries in pspwhich make previous fine-grained policy doesn't
work as expected.

Signed-off-by: Yu Chih <secret104278@gmail.com>

fredhaug committed:
[stable/prometheus] Add option to set labels for Prometheus AlertManager pods (#20026)

* Add option to set labels for Prometheus AlertManager pods.

Signed-off-by: Fredrik Haugseth <fredhaug93@gmail.com>

* Bump chart version.

Signed-off-by: Fredrik Haugseth <fredhaug93@gmail.com>

jonstelly committed:
[stable/openvpn] systemd-resolved, ip_forward, nodeport, ubuntu fixes (#16381)

* ipv4 forwarding init container

Signed-off-by: Kamil Szczygieł <kamil@szczygiel.io>

* typo

Signed-off-by: Kamil Szczygieł <kamil@szczygiel.io>

* openvpn: systemd-resolved, ip_forward, ubuntu fixes

    - routing / ip_forward issue and workaround
    - Ubuntu systemd-resolved and vpn gui issues
    - Move openvpn.conf to openvpn.serverConf for custom lines written to the server config file
    - Add openvpn.clientConf for custom lines written to the client config file
    - Major version bump for these breaking change
    - If chart is deployed using NodePort service, have clients connect to NodePort instead of externalPort

Signed-off-by: Jon Stelly <967068+jonstelly@users.noreply.github.com>

* Updates per PR comments/suggestions

Signed-off-by: Jon Stelly <967068+jonstelly@users.noreply.github.com>

* remove whitespace from values.yaml

Signed-off-by: Jon Stelly <967068+jonstelly@users.noreply.github.com>

* Add newlines

- Ensure newClientCert.sh contains newlines for each config option, prevents multiple openvpn directives from ending up on same line and being invalid

Signed-off-by: Jon Stelly <967068+jonstelly@users.noreply.github.com>

Co-authored-by: Kamil Szczygieł <kamil@szczygiel.io>

albertogeniola committed:
[stable/jenkins] (#20067)

* Implemented slaveListenerLoadBalancerIP

Signed-off-by: Alberto Geniola <albertogeniola@gmail.com>

* Bumped chart version

Signed-off-by: Alberto Geniola <albertogeniola@gmail.com>

* Check presence of slaveListenerLoadBalancerIP before assigningment

Signed-off-by: Alberto Geniola <albertogeniola@gmail.com>

kmjayadeep committed:
[stable/mongodb-replicaset] Added default values to missing keys in Values.yaml (#19924)

* Added default values to missing keys in mongodb-replicaset Values.yaml

Signed-off-by: Jayadeep KM <kmjayadeep@gmail.com>

* bump chart version

Signed-off-by: Jayadeep KM <kmjayadeep@gmail.com>

bashofmann committed:
[stable/nginx-ingress] Bump nginx-ingress version to 0.27.0 (#20069)

This also addresses the following bc breaks from the release notes (https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.27.0):

* Changed `controller.runAsUser` to `101`, due to a switch to alpinelinux
* Added `controller.maxmindLicenseKey` to enable download of the GeoLite2 database which is not included in the docker image anymore

Signed-off-by: Bastian Hofmann <bashofmann@gmail.com>

paprickar committed:
[stable/prometheus-pushgateway] Fix persistence support (#20048)

* Add persistence support

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* fix linting issue

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* Update Chart version

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* Avoid breaking existing behaviour

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* [stable/prometheus-pushgateway] Bump version (#16040)

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* [stable/prometheus-pushgateway] Fix persistence support

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

* Change back default persistence path

Signed-off-by: Patrick Kolze <patrick.kolze@doublecheck-it.de>

sergeimonakhov committed:
[stable/prometheus] Added remote_write and remote_read for prometheus (#20056)

* added remote_write and remote_read

Signed-off-by: Sergey Monakhov <sergey.monakhov@gmail.com>

* updated README.md and bumped chart version

Signed-off-by: Sergey Monakhov <sergey.monakhov@gmail.com>

* bump chart version

Signed-off-by: Sergey Monakhov <sergey.monakhov@gmail.com>

Commit:
[stable/phabricator] Release 9.0.5 updating components versions (#20071)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

bitfisher committed:
[stable/mariadb] inject `mariadb-replication-password` from secret on… (#19500)

* [stable/mariadb] inject `mariadb-replication-password` from secret only if replication is enabled

Signed-off-by: Markus Schmidleitner <markus.schmidleitner@ocilion.com>

* [stable/mariadb] increased chart version

Signed-off-by: Markus Schmidleitner <markus.schmidleitner@ocilion.com>

* [stable/mariadb] increased chart version

Signed-off-by: Markus Schmidleitner <markus.schmidleitner@ocilion.com>

* [stable/mariadb] bumped chart version

Signed-off-by: Markus Schmidleitner <markus.schmidleitner@ocilion.com>

lawliet89 committed:
[stable/prometheus] Add Telemetry port to Kube-state-metrics (#20072)

* [stable/prometheus] Add Telemetry port to Kube-state-metrics

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

* Bump kubestatemetrics to 1.9.1

Fixes SegFault with HPA

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

* Add Values for Service

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

zabudskyi committed:
[stable/owncloud] Set deployment strategy with values

Signed-off-by: Pavlo Zabudskyi <pavlo.zabudskyi@gmail.com>

sparqueur committed:
[stable/sonarqube] Add ability to add/mount extra volumes (#19364)

Signed-off-by: VALLEY Antoine <valley.antoine@gmail.com>
Signed-off-by: VALLEY Antoine (T0173847) <antoine.valley@mythalesgroup.io>

maruina committed:
Support custom sidecar containers (#20077)

Signed-off-by: Matteo Ruina <matteo.ruina@skyscanner.net>

RolandG committed:
[stable/stolon] add image.pullSecrets (#20088)

* [stable/stolon] add image.pullSecrets

Signed-off-by: Roland Gritzer <gritzer.roland@gmail.com>

* bumped chart version

Signed-off-by: Roland Gritzer <gritzer.roland@gmail.com>

seanson committed:
[stable/prometheus-operator] add runbook_url prefix (#19810) (#19811)

* [stable/prometheus-operator] add runbook_url prefix (#19810)

Fixes #19810

This adds support for parametrising runbook_url configuration in the
default rules configuration and the sync script. Some minor rules updates
as the latest changes are pulled in by the sync script.

Signed-off-by: Sean Johnson <seanson@users.noreply.github.com>

* [stable/prometheus-operator] chart version bump

Signed-off-by: Sean Johnson <seanson@users.noreply.github.com>

StupidScience committed:
stable/nginx-ingress/remove excessive - symbol (#20076)

Signed-off-by: StupidScience <tonysignal@gmail.com>

Eric-Fontana-Bose committed:
[stable/minio] Added support for ingress labels, which are required with some ingres… (#20092)

* Added support for ingress labels, which are required with some ingress controllers

Signed-off-by: Eric Fontana <eric_fontana@bose.com>

* Bump chart version

Signed-off-by: Eric Fontana <eric_fontana@bose.com>

paulhopkins11 committed:
Add additional values to enhance flexibility of helm-exporter (#20025)

* Add additional values to enhance flexibility of helm-exporter

Signed-off-by: Paul Hopkins <paulnewsoft@gmail.com>

* Remove namespace from metadata

Signed-off-by: Paul Hopkins <paulnewsoft@gmail.com>

Commit:
[stable/phpmyadmin] Release 4.2.7 updating components versions (#20095)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

hbagdi committed:
[stable/kong] 0.36.5 grpc protocol for plugin execution (#20099)

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

tylerauerbeck committed:
[incubator/hoverfly]: Add Openshift Route Creation for Admin & Proxy Interfaces (#20094)

* [incubator/hoverfly]: Add Openshift Route Creation for Admin & Proxy Interfaces

Signed-off-by: Tyler Auerbeck <tauerbec@redhat.com>

* [incubator/hoverfly]: Add Openshift Route Creation for Admin & Proxy Interfaces; Bump Chart Version

Signed-off-by: Tyler Auerbeck <tauerbec@redhat.com>

* [incubator/hoverfly]: Add Openshift Route Creation for Admin & Proxy Interfaces; Bump Chart Version; Fix Linting

Signed-off-by: Tyler Auerbeck <tauerbec@redhat.com>

LucasBoisserie committed:
Limit Alerts Scope (#19538)

Signed-off-by: LucasBoisserie <lucas.boisserie@gmail.com>

maruina committed:
Bump kube-state-metrics version (#19874)

Signed-off-by: Matteo Ruina <matteo.ruina@skyscanner.net>

alemorcuq commented on stable/owncloud/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: owncloud
-version: 8.0.1
+version: 8.0.2
```
The version should be a new minor.

alemorcuq commented on stable/owncloud/values.yaml:
```diff
@@ -218,6 +218,9 @@ persistence:
 
+# Update strategy type (e.g. Recreate or RollingUpdate)
+updateStrategy: RollingUpdate
+
```
I would do this instead, allowing for more customization:

```yaml
## Set up update strategy for the ownCloud installation.
## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
## Example:
# updateStrategy:
#  type: RollingUpdate
#  rollingUpdate:
#    maxSurge: 25%
#    maxUnavailable: 25%
updateStrategy:
  type: RollingUpdate
```

alemorcuq commented on stable/owncloud/README.md:
```diff
@@ -95,6 +95,7 @@ The following table lists the configurable parameters of the ownCloud chart and
 | `persistence.owncloud.existingClaim`| An Existing PVC name for ownCloud volume   | `nil` (uses alpha storage class annotation)             |
 | `persistence.owncloud.accessMode`   | PVC Access Mode for ownCloud volume        | `ReadWriteOnce`                                         |
 | `persistence.owncloud.size`         | PVC Storage Request for ownCloud volume    | `8Gi`                                                   |
+| `updateStrategy`                    | Owncloud deployment strategy               | `RollingUpdate`                                         |
```
I would change the default value to:

```Markdown
`{type: "RollingUpdate"}`
```

See the other comments.

alemorcuq commented on stable/owncloud/templates/deployment.yaml:
```diff
@@ -14,6 +14,8 @@ spec:
       release: "{{ .Release.Name }}"
   replicas: 1
+  strategy:
+    type: {{ .Values.updateStrategy }}
```
I would do this instead, allowing for more customization:

```yaml
{{- if .Values.updateStrategy }}
strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
{{- end }}
```

wilmardo committed:
fix: do not fail user-create container when user already exists (#20106)

Signed-off-by: wilmardo <info@wilmardenouden.nl>

usmonster committed:
[stable/sonarqube] upgrades LTS to latest patch (#19836)

* [stable/sonarqube] upgrades LTS to latest patch

Also updates ingress apiVersion, adding support for later k8s versions.
(Modernizes documentaiton accordingly.)

Signed-off-by: Usman Akeju <akeju00+github@gmail.com>

* [stable/sonarqube] support also k8s <= v1.13

Signed-off-by: Usman Akeju <akeju00+github@gmail.com>

cablespaghetti committed:
[incubator/kafka] Add Prometheus Operator PrometheusRules (#20110)

* [incubator/kafka] Add Prometheus Operator PrometheusRules

Signed-off-by: Sam Weston <weston.sam@gmail.com>

* [incubator/kafka] Further README.md updates

Signed-off-by: Sam Weston <weston.sam@gmail.com>

mrueg committed:
[stable/kube-state-metrics] Version bump to 1.9.2 (#20103)

Signed-off-by: Manuel Rüger <manuel@rueg.eu>

VladyslavTokar committed:
[stable/elasticsearch-exporter] Enabling environment from secrets (#19115)

* [stable/elasticsearch-exporter] Enabling environment from secrets

Signed-off-by: VladyslavTokar <g.vtokar@gmail.com>

* [stable/elasticsearch-exporter] Update description for envFromSecret parameter in readme

Signed-off-by: VladyslavTokar <g.vtokar@gmail.com>

hbagdi committed:
[stable/kong] 0.36.6 deprecate Kong chart (#20149)

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

Commit:
[stable/external-dns] Release 2.14.3 updating components versions (#20133)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/mongodb] Release 7.6.5 updating components versions (#20123)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/phpmyadmin] Release 4.2.8 updating components versions (#20141)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/prestashop] Release 9.1.2 updating components versions (#20150)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/ghost] Release 9.1.2 updating components versions (#20154)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/joomla] Release 7.1.3 updating components versions (#20120)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/testlink] Release 7.0.3 updating components versions (#20130)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/moodle] Release 7.0.7 updating components versions (#20143)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/kubewatch] Release 1.0.3 updating components versions (#20131)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/orangehrm] Release 7.0.3 updating components versions (#20125)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/suitecrm] Release 8.0.3 updating components versions (#20126)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/phpbb] Release 7.0.3 updating components versions (#20128)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/postgresql] Release 8.1.4 updating components versions (#20129)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/rabbitmq] Release 6.16.2 updating components versions (#20134)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/redis] Release 10.3.3 updating components versions (#20136)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/redmine] Release 14.1.3 updating components versions (#20138)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/jasperreports] Release 7.0.3 updating components versions (#20139)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/mediawiki] Release 9.1.2 updating components versions (#20140)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/mariadb] Release 7.3.4 updating components versions (#20142)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/nats] Release 4.2.9 updating components versions (#20145)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/parse] Release 10.3.2 updating components versions (#20147)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/opencart] Release 7.0.3 updating components versions (#20155)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

divad4686 committed:
[stable/prometheus-operator] Update grafana chart version (#20096)

* Update grafana chart version

Signed-off-by: David Martinez <divad4686@gmail.com>

* Update chart version

Signed-off-by: David Martinez <divad4686@gmail.com>

* Recreate requirements.lock

Signed-off-by: David Martinez <divad4686@gmail.com>

Commit:
[stable/dokuwiki] Release 6.0.4 updating components versions (#20122)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/odoo] Release 12.0.6 updating components versions (#20124)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/phabricator] Release 9.0.6 updating components versions (#20135)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/wordpress] Release 8.1.1 updating components versions (#20137)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/drupal] Release 6.2.2 updating components versions (#20144)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/owncloud] Release 8.0.3 updating components versions (#20152)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/osclass] Release 7.0.3 updating components versions (#20153)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/nats] Fix args for metrics (#20159)

Signed-off-by: juan131 <juan@bitnami.com>

Commit:
[stable/phpmyadmin] Fix an SSL certificates directory conflict (#20086)

Mount the SSL certificates used to connect to the database in  /db_certs instead of /certs, as /certs is already used by the bitnami/phpmyadmin Docker image to configure Apache internally.

Signed-off-by: Nicolas Camus <nicolas@octopulse.io>

zabudskyi committed:
[stable/owncloud] Set deployment strategy with values

Signed-off-by: Pavlo Zabudskyi <pavlo.zabudskyi@gmail.com>
